src/controllers/controllermanager.cpp

```diff
@@ -37,6 +37,12 @@ const int kPollIntervalMillis = 5;
 const int kPollIntervalMillis = 1;
 #endif
 
+// Note: A standard Midi device runs at 31.25 kbps,
+// a usual Midi message has 30 bits so we may not expect more than
+// 1000 messages per second
+// PortMidiController uses a buffer of 64 messages which results in
+// Maximum polling interval of 64 ms a theoritical minimum of 1 ms
+
```

Comment:
Yeah I agree with @Pegasus-RPG that high-end USB midi devices may not conform to the standard assumptions about midi rates.  My VCI400 is 4096 ticks per revolution and I can get 3 loops through this code by spinning the wheel really fast.  So if we set a max of 5 that seems safe to me.